conjur/user/user_logic.py

```diff
@@ -35,36 +35,38 @@ def rotate_api_key(self, resource_to_update):
         2. Rotate another users API key
         """
         loaded_credentials = self.extract_credentials_from_credential_store()
```

Comment:
I think `credentials` is enough
```suggestion
        credentials = self.extract_credentials_from_credential_store()
```

Reply:
you can prefix them with `current_` or `logged_in_` if you want, for clarity.